tailormap-components/projects/core/src/lib/user-interface/attributelist/attributelist-common/attributelist-datasource.ts

```diff
@@ -143,6 +143,18 @@ export class AttributeDataSource extends DataSource<any> {
     return cnt;
   }
 
+  public getfirstRowAsAttributeListFeature(): AttributeListFeature[] {
+    const features: AttributeListFeature[] = [];
+    this.rows.forEach( (row: RowData) => {
+      features.push({
+        features: row,
+        related_featuretypes: row.related_featuretypes,
+        __fid: row.__fid,
+      });
+    });
```

Comment:
De functie heeft delen hetzelfde als getCheckedRowsAsAttributeListFeature: het vertalen van een rij naar een feature. In een helper stoppen?